aikau/src/main/resources/alfresco/forms/Form.js

```diff
@@ -384,6 +405,24 @@ define(["dojo/_base/declare",
 
             this.processWidgets(this.widgets, this._form.domNode, "FIELDS");
          }
+
+         // If configured, focus the first field in the form
+         if (this.firstFieldFocusOnLoad)
+         {
+            this.focus();
+         }
+
+         // If configured, allow the form to be submitted by pressing the enter key
+         if (this.submitOnEnter && !this.autoSavePublishTopic)
+         {
+            on(this, "keydown", lang.hitch(this, function(event){
+               if (event.keyCode === dojo.keys.ENTER && this.invalidFormControls.length === 0)
+               {
+                  var payload = lang.mixin(this.okButtonPublishPayload || {}, this.getValue());
```

Comment:
This would be better as shared code, e.g. maybe actioning the confirmation button directly or getting it's payload rather than duplicating code to create the payload.